### DISCUSSION TITLE
Document to understand Explain analyze verbose command

### DISCUSSION THREAD

**Discussion by atifiu** [trinodb/trino#17942] [Q&A]
Hi guys,

I am working on a project to integrate Iceberg and Trino and also doing performance comparisons between processing/reading/writing iceberg tables using Spark and Trino. So to better understand how many queries are performing I need to understand the explain analyze verbose command output. So is there any documentation available ? Or how can i read it in a better way ?

**Comment by findinpath** [comment]
This area is not documented in Trino indeed.

However, you could make use of https://github.com/trinodb/trino/pull/16950 to see in detail what analysis, planning, execution phases involve while running the query.
